ports/nrf/boards/pca10059/readme.md

```diff
@@ -0,0 +1,7 @@
+# Pins mapping
+
+Software Pins 0-31 correspond to physical pins 0.x and software Pins 32-63
```

Comment:
I would suggest to change "... software Pins 32-63" to "... software Pins 32-48" as nrf52840 only provides 48 GPIO's. (The same is the case for the nrf53 device which also have 48 GPIOs).

Reply:
Done.

Reply:
I'm so sorry!
Terrible mistake from my side. Of course should would be 32 - **47**, not 48.

Reply:
No problem. I addressed it